src/SFA.DAS.Employer.PR.Web/Controllers/YourTrainingProvidersController.cs

```diff
@@ -52,12 +50,12 @@ private YourTrainingProvidersViewModel PopulateYourTrainingProvidersViewModel(st
             {
                 addedUkprns.Add(permissions.Ukprn);
 
-                var outstandingRequest = legalEntity.Requests.Find(a => a.Ukprn == permissions.Ukprn);
+                var outstandingRequest = legalEntity.Requests.Find(a => a.Ukprn == permissions.Ukprn && a.RequestType != RequestType.CreateAccount);
```

Comment:
Im having trouble understanding why a filter for CreateAccount is added here.  Please just confirm that is correct, I'm happy to take your word for it, but I just wanted to be sure.